{
    "source": "BDSA",
    "name": "BDSA-2020-3921",
    "title": "Pure-FTPd Vulnerable to Denial-of-Service (DoS) via New Server Connections",
    "description": "Pure-FTPd is vulnerable to denial-of-service (DoS) due to a resource exhaustion issue. An attacker could exploit this vulnerability by supplying a application with a mass of connections, in order to cause excessive consumption of resources.",
    "technicalDescription": "This vulnerability exists due to Pure-FTPd only allowing a certain amount of connections to be made to the server. This means that an attacker could repeatedly make new connections to the server which could prevent legitimate users from connecting.",
    "publishedDate": "2020-12-30T10:00:50.196Z",
    "updatedDate": "2022-05-30T09:00:06.477Z",
    "disclosureDate": "2020-11-26T00:00:00.000Z",
    "exploitPublishDate": "2020-11-26T00:00:00.000Z",
    "solution": "",
    "severity": "MEDIUM",
    "cvss2": {
        "baseScore": 5.0,
        "impactSubscore": 2.9,
        "exploitabilitySubscore": 10.0,
        "severity": "MEDIUM",
        "accessVector": "NETWORK",
        "accessComplexity": "LOW",
        "authentication": "NONE",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "PARTIAL",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "UNCONFIRMED",
            "score": 4.0
        },
        "vector": "(AV:N/AC:L/Au:N/C:N/I:N/A:P/E:POC/RL:U/RC:UC)"
    },
    "cvss3": {
        "baseScore": 5.3,
        "impactSubscore": 1.4,
        "exploitabilitySubscore": 3.9,
        "severity": "MEDIUM",
        "attackVector": "NETWORK",
        "attackComplexity": "LOW",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "availabilityImpact": "LOW",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "temporalMetrics": {
            "exploitability": "PROOF_OF_CONCEPT",
            "remediationLevel": "UNAVAILABLE",
            "reportConfidence": "UNKNOWN",
            "score": 4.6
        },
        "vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:L/E:P/RL:U/RC:U"
    },
    "useCvss3": true,
    "classifications": [],
    "zeroDay": false,
    "underReview": false,
    "parentAdvisory": false,
    "credit": "xynmaps",
    "workaround": "",
    "bdsaTags": [],
    "overallScore": 4.6,
    "_meta": {
        "allow": [
            "GET"
        ],
        "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2020-3921",
        "links": [
            {
                "rel": "cwes",
                "href": "https://trendmicro2.app.blackduck.com/api/cwes/CWE-400"
            },
            {
                "rel": "cwes",
                "href": "https://trendmicro2.app.blackduck.com/api/cwes/CWE-770"
            },
            {
                "rel": "related-vulnerability",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/CVE-2020-35359",
                "label": "NVD"
            },
            {
                "rel": "bdsa-ranges",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2020-3921/ranges"
            },
            {
                "rel": "reference",
                "href": "https://www.exploit-db.com/exploits/49105",
                "label": "ADVISORY"
            },
            {
                "rel": "reference",
                "href": "https://www.exploit-db.com/exploits/49105",
                "label": "POC"
            },
            {
                "rel": "default-remediation-status",
                "href": "https://trendmicro2.app.blackduck.com/api/vulnerabilities/BDSA-2020-3921/default-remediation-status"
            }
        ]
    }
}